test/sphere_sphericalharmonics.jl

```diff
@@ -0,0 +1,59 @@
+# using Legendre
+# # input lat and long in degrees
+
+# function Ylm(l,m,lat,long) 
+# 	return real(
+# 		λlm(l,m,sind(lat)) * exp(im * m * deg2rad(long)),
+# 	)
+# end
```

Comment:
I would remove dead code. Any reason this wanted to be preserved?

Reply:
read her comment, it's a stub for when the package is registered

Reply:
ok, I didn't see any comment here